setup.py

```diff
@@ -16,9 +16,9 @@
 info = dict(
     CodaAuthorString = 'Justin Hileman',
     CodaDescriptionString = 'Generate PHP Docblocks like magick!',
-    CFBundleVersion = '2.0.1',
+    CFBundleVersion = '2.0.2',
```

Comment:
Let's leave the bundle version the same and I'll bump it when I tag the release.